### ISSUE TITLE
Multi-Class Classification with Probalities

### ISSUE THREAD

**Issue by sudheerpunuri** [keras-team/keras#17079]
Dear All:
Can anyone suggest me a Multi-class classifier which give probabilities.  By using these probability I wanted to rank the classes.

**Comment by tilakrayal** [comment]
@sudheerpunuri,
Could you please elaborate about your Feature. Also, please specify the Use Cases for this feature. Thank you!

**Comment by sudheerpunuri** [comment]
I need a classifier for multiclass classification. I have 7 classes in my dataset. The classifier must give the probabilities. Pls suggest.

**Comment by MominAhmedShaikh** [comment]
Make a Simple CNN model with **softmax** as an activation in the last dense layer with no. of output as 7.
It will give you probabilities of the class predicted and later on you can use those probabilities to rank your classes.

You can refer to this repository where i had **50 classes** to predict.

[https://github.com/MominAhmedShaikh/CNN-Multiclass-classification-BarkVNet-50](url)

**Comment by tilakrayal** [comment]
@sudheerpunuri,
Could you please take a look at this [comment](https://github.com/keras-team/keras/issues/17079#issuecomment-1262040468) from the user & refer to the doc [link](https://github.com/MominAhmedShaikh/CNN-Multiclass-classification-BarkVNet-50) and try to test the code on a new virtual environment. Thank you!
